src/components/Message/MessageStatus.tsx

```diff
@@ -57,15 +59,15 @@ const UnMemoizedMessageStatus = <
     );
   }
 
-  if (readBy?.length && !threadList && !justReadByMe) {
-    const lastReadUser = readBy.filter((item) => item.id !== client.user?.id)[0];
+  if (filteredReadBy.length && !threadList && !justReadByMe) {
+    const [lastReadUser] = filteredReadBy.filter((item) => item.id !== client.user?.id);
 
     return (
       <span
         className={`str-chat__message-${messageType}-status`}
         data-testid='message-status-read-by'
       >
-        <Tooltip>{getReadByTooltipText(readBy, t, client)}</Tooltip>
+        <Tooltip>{getReadByTooltipText(filteredReadBy, t, client)}</Tooltip>
         <Avatar
           image={lastReadUser.image}
           name={lastReadUser.name || lastReadUser.id}
```

Comment:
Can we add the guards here just in case? `lastReadUser?.image` and `lastReadUser?.name`

I know its not required anymore ... but to avoid the hassle regardless in case the issue persists